Gemfile

```diff
@@ -18,7 +18,7 @@ group :test do
   gem "capybara"
   gem "evergreen"
   gem "capybara-webkit"
-  gem 'selenium-webdriver'
+  gem 'selenium-webdriver', '2.53.4'
```

Comment:
I tried using the latest Firefox but that also seems to be broken with the latest selenium driver for Ruby. I can look more into this next week probably. 

OTOH I can also revert to the previous commit which uses selenium-webdriver 2.53.4, In fact I will prefer this b/c on my system I have an older Firefox version (for now).